src/dotnet-install.ps1

```diff
@@ -619,6 +619,23 @@ function DownloadFile($Source, [string]$OutPath) {
     }
 }
 
+function SafeRemoveFile($Path) {
+    try {
+        if (Test-Path $Path) {
+            Remove-Item $Path
+            Say-Verbose "The temporary file `"$Path`" was removed."
+        }
+        else
+        {
+            Say-Verbose "The temporary file `"$Path`" does not exist, therefore is not removed."
+        }
+    }
+    catch
+    {
+        Say "Failed to remove the temporary file: `"$Path`", remove it manually."
```

Comment:
It would be nice if we have a function "Say-Warning" or even "Say-Error". I'd rather like to show this message in a different color, to make it more visible.

Reply:
I would agree, but would prefer to do it in different issue/PR.  There is a number of messages in the scripts that are warnings/errors, but are using Say for that.

Reply:
Let's do this in other PR